nncf/quantization/algorithms/min_max/torch_backend.py

```diff
@@ -158,19 +156,27 @@ def unify_statistics(statistics: List[PTMinMaxTensorStatistic]) -> PTMinMaxTenso
         min_values = torch.amin(torch.stack(min_values), dim=0)
         return PTMinMaxTensorStatistic(min_values=min_values, max_values=max_values)
 
+    @staticmethod
+    def get_target_point_shape(nncf_graph: NNCFGraph, node: NNCFNode, target_point: PTTargetPoint) -> List[int]:
+        if target_point.is_weight_target_point():
+            return node.layer_attributes.get_weight_shape()
+        return nncf_graph.get_input_shape_for_insertion_point(target_point)
+
+    @staticmethod
+    def get_channel_axes(node: NNCFNode, target_point: PTTargetPoint) -> Tuple[int]:
+        if target_point.is_weight_target_point():
+            return (node.layer_attributes.get_target_dim_for_compression(),)
+        return (1,)
```

Comment:
The methods `get_target_point_shape` and `get_channel_axes` have been introduced to provide functionality for determining the shape of the target point and the axes for channel-wise operations, respectively. Both methods are well-structured and follow the static method pattern, which is consistent with the rest of the class. However, it's important to ensure that these methods are covered by unit tests to validate their behavior in various scenarios, especially edge cases. Do you have tests for these functions?

Reply:
Test was added